### ISSUE TITLE
Make sure it can be easily be integrated with other projects

### ISSUE THREAD

**Issue by SantiagoPittella** [lambdaclass/madara_explorer#260]
- check that it is easy to use other data sources, that we don't rely on the feeder gateway

**Comment by 9oelM** [comment]
Hey @SantiagoPittella [the feeder gateway will be deprecated within this year](https://docs.starknet.io/documentation/starknet_versions/upcoming_versions/). Any plans to remove the reliance on the feed gateway rather than growing reliance on it? This is not only for integrating with other projects but also this repository itself, because you are not going to be able to use it in Nov or Dec this year anyway

**Comment by SantiagoPittella** [comment]
Yes, we are ready to switch to full RPC when the nodes support the features that we use from the feeder gateway. We are looking forward to it, since it also will improve the performance of the explorer.
